packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.tsx

```diff
@@ -54,24 +73,36 @@ const ProjectExtensionProviderForImageRegex = (props: ProviderProps) => {
 ProjectExtensionProviderForImageRegex.displayName =
   'ProjectExtensionProviderForImageRegex';
 
-const GetProjectExtensionImageRegex = (props: ConsumerProps) => (
-  <Context.Consumer>
-    {(imageRegexContext) => props.render(imageRegexContext)}
-  </Context.Consumer>
-);
+const GetProjectExtensionImageRegex = (props: ConsumerProps) => {
+  //TODO: add a descriptive possible deprecation warning for moving from HOC to Hooks in the future
```

Comment:
This would be my suggestion:
```
@commercetools-frontend/application-shell-connectors: It is not recommended to use the 'GetProjectExtensionImageRegex' anymore. Please use the 'useProjectExtensionImageRegex' hook instead.
```